docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -1,11 +1,9 @@
 ---
 title: Models - Strapi Developer Documentation
-description: (…)
+description: As Strapi is a headless Content Management System (CMS), creating a data structure for the content is one of the most important aspects of using the software. Strapi models (i.e. content-types, components, and dynamic zones) define a representation of the data structure.
```

Comment:
This description is too long, it would be nice to not exceed 160 characters :)

Reply:
```suggestion
description: Strapi models (i.e. content-types, components, and dynamic zones) define a representation of the data structure.
```